### ISSUE TITLE
Give opencve the capability to run a 'ondemand' import

### ISSUE THREAD

**Issue by SPoint42** [opencve/opencve#140]
OpenCVE update is great but sometime you could want to launch an import of the database manually. 
Give OpenCVE the capability to do a `opencve update-cve` for example

**Comment by ncrocfer** [comment]
I don't understand what you mean by "launching an import of the database manually".

This is exactly the goal of the Celery tasks to automatically update the database, without having to worry about doing it manually.

Why do you want to do it manually? What is your usecase?

**Comment by SPoint42** [comment]
Yes, but I stop the docker sometime and don't want to wait too much time for the celery to run. So If I could launch manually the import, this could be great

**Comment by M0dEx** [comment]
I'll add to this a bit of my experience.

One instance I am the administrator of has stopped importing CVEs for a while before I noticed (~2 weeks). After I noticed, I restarted it and everything seemed to be working and new CVEs were imported correctly. However, some CVEs from the 2 week gap were not imported and apart from full wipe and complete re-import, I have no way of importing the to the instance.

This would be of great help in such cases!

**Comment by ncrocfer** [comment]
Closing this issue as it has been inactive for a long time (BTW OpenCVE 2 is now based on Airflow, so users only have to re-run the wanted DAGruns).
